packages/cli/sls/deploy.js

```diff
@@ -51,8 +51,8 @@ module.exports = async inputs => {
                         `🚨 Environment ${green(env)} is missing in ${green(
                             app.location + "/.env.json"
                         )} file.`,
-                        "Have you deployed the corresponding API environment?.",
-                        'To deploy the API, run "webiny deploy-api --env=dev".'
+                        "Have you deployed the corresponding API environment?",
+                        `To deploy the API, run ${blue('webiny deploy-api --env=dev')}.`
```

Comment:
This message should actually include the working command, so user can C/P it and run, so please make it `--env=${env}` so we get an actual environment that is missing in the command. Thanks!

Reply:
oh yes, i forgot that the env is dynamic, sorry 😅